plugins/rbac/src/api/RBACBackendClient.ts

```diff
@@ -247,4 +252,26 @@ export class RBACBackendClient implements RBACAPI {
     }
     return jsonResponse;
   }
+
+  async deletePolicies(entityReference: string, policies: Policy[]) {
+    const { token: idToken } = await this.identityApi.getCredentials();
+    const backendUrl = this.configApi.getString('backend.baseUrl');
+    const { kind, namespace, name } = getKindNamespaceName(entityReference);
+    const jsonResponse = await fetch(
+      `${backendUrl}/api/permission/policies/${kind}/${namespace}/${name}`,
+      {
+        headers: {
+          ...(idToken && { Authorization: `Bearer ${idToken}` }),
+          'Content-Type': 'application/json',
+          Accept: 'application/json',
+        },
+        body: JSON.stringify(policies),
+        method: 'DELETE',
+      },
+    );
+    if (jsonResponse.status !== 200 && jsonResponse.status !== 204) {
+      return { status: 'Failed', statusText: jsonResponse.statusText };
+    }
+    return 'Successful';
```

Comment:
do we need to return custom string? can't we return jsonResponse and let the caller handle it

Reply:
Yep, or even just `void`.  It could also simplify code calling this function to throw the error instead of returning this  error object, as this I guess would normally be an exceptional case.